src/main/java/com/google/cloud/tools/managedcloudsdk/install/Installer.java

```diff
@@ -63,6 +87,12 @@ public void install()
     command.add("--quiet"); // don't accept user input during install
     command.add("--usage-reporting=" + usageReporting); // usage reporting passthrough
 
+    if (overrideComponents != null) {
```

Comment:
or size != 0

Reply:
running the installation script actually does work if no components are specified. if you just do `install.sh --override-components`, no components will be installed. this seems like something we would want to allow